test/test_distributed.py

```diff
@@ -1357,6 +1374,86 @@ def test_DistributedDataParallel(self):
         gpus = list(map(lambda i: torch.device('cuda:' + str(i)), gpus))
         self._test_DistributedDataParallel(gpu_subset=gpus, rank=rank, output_device=torch.device('cuda'))
 
+    def _convert_syncbn_model(self, module, process_group=None):
+        mod = module
+        if isinstance(module, nn.modules.batchnorm._BatchNorm):
+            mod = nn.SyncBatchNorm(
+                module.num_features,
+                module.eps,
+                module.momentum,
+                module.affine,
+                module.track_running_stats,
+                process_group
+            )
+            mod.running_mean = module.running_mean
+            mod.running_var = module.running_var
+            if module.affine:
+                mod.weight.data = module.weight.data.clone().detach()
+                mod.bias.data = module.bias.data.clone().detach()
+        for name, child in module.named_children():
+            mod.add_module(name, self._convert_syncbn_model(child))
+        del module
+        return mod
```

Comment:
The mod vs module naming is confusing. Input/output or something else like it is more appropriate IMO.

Reply:
will rename. Thanks.